vhdbuilder/packer/vhd-scanning-exe-on-vm.sh

```diff
@@ -45,14 +47,18 @@ else
 fi
 
 # trivy scan must have run before this
-az storage blob upload --file /opt/azure/containers/trivy-report.json \
+az storage blob upload --file /opt/azure/containers/trivy-report-rootfs.json \
     --container-name ${SIG_CONTAINER_NAME} \
-    --name ${TRIVY_REPORT_NAME} \
+    --name ${TRIVY_REPORT_ROOTFS_NAME} \
```

Comment:
can we avoid the extra var and align the setting to be in place like you did for the blob upload below ?

`--name ${IMAGE_REPORT}-${BUILD_ID}-${TIMESTAMP}.json`